### PR TITLE
Add LICENSE files to sdists

### LINKED ISSUE #4247: LICENSE file missing in pypi tarball
The source tarball https://pypi.io/packages/source/m/mypy/mypy-0.550.tar.gz doesn't have the LICENSE file. Please add it to the manifest.

### PR DESCRIPTION
**Pull request by ethanhs**
Fixes #4247

### CODE REVIEW

File: extensions/LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Mypy extensions are licensed under the terms of the MIT license, reproduced below.
+
+= = = = =
+
+The MIT License
+
+Copyright (c) 2015-2016 Jukka Lehtosalo and contributors
```
Comment (ilevkivskyi) [inline extensions/LICENSE]: Shouldn't this be updated to year 2017?

Comment (ethanhs) [reply extensions/LICENSE]: AIUI, from what I have read updating it is not needed, but I will go ahead and make that change if you want.

Comment (ilevkivskyi) [reply extensions/LICENSE]: I am not an expert myself, but this is what I have seen typically.

Comment (ethanhs) [reply extensions/LICENSE]: Updated.

**Pull request merged**